spec/ruby/core/io/timeout_spec.rb

```diff
@@ -0,0 +1,104 @@
+# -*- encoding: utf-8 -*-
+require_relative '../../spec_helper'
+
+describe "IO#timeout" do
```

Comment:
This spec should also test the successful case, i.e., with a timeout big enough to allow an operation to complete.

Reply:
Added.